hack/build.sh

```diff
@@ -9,8 +8,27 @@ BUILD_DATE=$(date --utc -Iseconds)
 
 mkdir -p bin
 
-LDFLAGS="-s -w "
-LDFLAGS+="-X github.com/medik8s/node-maintenance-operator/version.Version=${VERSION} "
-LDFLAGS+="-X github.com/medik8s/node-maintenance-operator/version.GitCommit=${COMMIT} "
-LDFLAGS+="-X github.com/medik8s/node-maintenance-operator/version.BuildDate=${BUILD_DATE} "
-GOFLAGS=-mod=vendor CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="${LDFLAGS}" -o bin/manager main.go
+LDFLAGS_VALUE="-X github.com/medik8s/node-maintenance-operator/version.Version=${VERSION} "
+# additional values must be quoted when used with GOFLAGS!
```

Comment:
- you don't do what the comment says
- it still works though... probably because of the single quotes later on
- so please remove this comment